guides/components/component-arguments-and-html-attributes.md

```diff
@@ -1,144 +1,115 @@
 
 <div class="cta">
   <div class="cta-note">
     <div class="cta-note-body">
-      <div class="cta-note-heading">Zoey says...</div>
+      <div class="cta-note-heading">Zoey dit...</div>
       <div class="cta-note-message">
         <p>
-          You might be wondering why Ember uses the `@` syntax for its
-          components instead of normal HTML attribute syntax. We'll learn why
-          in the next section.
+          Vous vous demandez peut-être pourquoi Ember utilise la syntaxe `@` pour ses composants au lieu de la syntaxe normale des attributs HTML. Nous apprendrons pourquoi dans la section suivante.
         </p>
       </div>
     </div>
     <img src="/images/mascots/zoey.png" role="presentation" alt="">
   </div>
 </div>
 
-## HTML Attributes
+## Attributs HTML
 
-Let's try to use our `<Avatar>` component for the sent message avatar.
+Essayons d'invoquer notre composant `<Avatar>` pour l'avatar du message envoyé.
 
 ```handlebars {data-filename="app/components/sent-message/avatar.hbs"}
-<Avatar @title="Zoey's avatar" @initial="Z" />
+<Avatar @title="Avatar de Zoey" @initial="Z" />
 ```
 
-We're really, really close.
+Nous y sommes presque.
 
 ```handlebars {data-filename="output" data-diff="-1,+2"}
 <aside class="current-user">
 <aside>
-  <div class="avatar" title="Zoey's avatar">Z</div>
+  <div class="avatar" title="Avatar de Zoey">Z</div>
 </aside>
 ```
 
-We're just missing the `current-user` class on the HTML `<aside>` element. To
-make that work, we'll specify the HTML attribute `class` on the `<Avatar>` tag.
+Il nous manque juste la classe `current-user` sur l'élément HTML `<aside>`. Pour l'ajouter, nous allons spécifier l'attribut HTML `class` sur la balise `<Avatar>`.
 
 ```handlebars {data-filename="app/components/sent-message/avatar.hbs"}
 <Avatar
-  @title="Zoey's avatar"
+  @title="Avatar de Zoey"
   @initial="Z"
   class="current-user"
 />
 ```
 
-The avatar component also needs to specify where to put attributes that were
-specified on the tag.
+Le composant `avatar` doit également spécifier où placer les attributs qui ont été passés à la balise.
 
 ```handlebars {data-filename="app/components/avatar.hbs"}
 <aside ...attributes>
   <div class="avatar" title={{@title}}>{{@initial}}</div>
 </aside>
 ```
 
-The `...attributes` syntax determines where the attributes from a tag should
-appear in the component's template. Any number of attributes can be specified on
-the avatar component now, and they will all end up on the element that has
-`...attributes`.
+La syntaxe `...attributes` détermine où les attributs d'une balise doivent apparaître dans le _template_ du composant. N'importe quel nombre d'attributs peuvent désormais être spécifiés sur composant `avatar`, et ils se retrouveront tous sur l'élément qui a la syntaxe `...attributes`.
```

Comment:
```suggestion
La syntaxe `...attributes` détermine où les attributs d'une balise doivent apparaître dans le _template_ du composant. N'importe quel nombre d'attributs peuvent désormais être spécifiés sur composant `<Avatar>`, et ils se retrouveront tous sur l'élément qui a la syntaxe `...attributes`.
```

Unlike the English version, I think we should specify the name of the syntax `...attribute`, which would benefit the reader:

```suggestion
La syntaxe `...attributes` (appelée _splattributes_) détermine où les attributs d'une balise doivent apparaître dans le _template_ du composant. N'importe quel nombre d'attributs peuvent désormais être spécifiés sur composant `<Avatar>`, et ils se retrouveront tous sur l'élément qui a la syntaxe `...attributes`.
```